src/vivarium_census_prl_synth_pop/components/household.py

```diff
@@ -135,13 +166,23 @@ def update_household_addresses(self, household_ids: pd.Series) -> None:
             len(household_ids), taken=self._households["address_id"]
         )
         po_boxes = self.generate_po_boxes(len(household_ids))
+        states_pumas = self.generate_states_pumas(household_ids)
         self._households.loc[household_ids, "address_id"] = new_address_ids
         self._households.loc[household_ids, "po_box"] = po_boxes
+        self._households.loc[household_ids, ["state_id", "puma"]] = states_pumas
 
     ##################
     # Helper methods #
     ##################
 
+    @staticmethod
+    def _get_state_puma_options(df: pd.DataFrame) -> pd.Series:
```

Comment:
Could this method not be replaced with `df[["state", "puma"]].drop_duplicates()`? I think it will even be cleaner below to have it as a DataFrame instead of a list when using `vectorized_choice`.